replay/replay.go

```diff
@@ -94,45 +95,45 @@ func Replay(t *testing.T, server pulumirpc.ResourceProviderServer, jsonLog strin
 	switch entry.Method {
 
 	case "/pulumirpc.ResourceProvider/GetSchema":
-		replay(t, entry, new(pulumirpc.GetSchemaRequest), server.GetSchema)
+		replay(t, entry, new(pulumirpc.GetSchemaRequest), server.GetSchema, nil)
 
 	case "/pulumirpc.ResourceProvider/CheckConfig":
-		replay(t, entry, new(pulumirpc.CheckRequest), server.CheckConfig)
+		replay(t, entry, new(pulumirpc.CheckRequest), server.CheckConfig, normCheckResponse)
 
 	case "/pulumirpc.ResourceProvider/DiffConfig":
-		replay(t, entry, new(pulumirpc.DiffRequest), server.DiffConfig)
+		replay(t, entry, new(pulumirpc.DiffRequest), server.DiffConfig, nil)
 
 	case "/pulumirpc.ResourceProvider/Configure":
-		replay(t, entry, new(pulumirpc.ConfigureRequest), server.Configure)
+		replay(t, entry, new(pulumirpc.ConfigureRequest), server.Configure, nil)
 
 	case "/pulumirpc.ResourceProvider/Invoke":
-		replay(t, entry, new(pulumirpc.InvokeRequest), server.Invoke)
+		replay(t, entry, new(pulumirpc.InvokeRequest), server.Invoke, normInvokeResponse)
 
 	// TODO StreamInvoke might need some special handling as it is a streaming RPC method.
 
 	case "/pulumirpc.ResourceProvider/Call":
-		replay(t, entry, new(pulumirpc.CallRequest), server.Call)
+		replay(t, entry, new(pulumirpc.CallRequest), server.Call, normCallResponse)
 
 	case "/pulumirpc.ResourceProvider/Check":
-		replay(t, entry, new(pulumirpc.CheckRequest), server.Check)
+		replay(t, entry, new(pulumirpc.CheckRequest), server.Check, normCheckResponse)
 
 	case "/pulumirpc.ResourceProvider/Diff":
-		replay(t, entry, new(pulumirpc.DiffRequest), server.Diff)
+		replay(t, entry, new(pulumirpc.DiffRequest), server.Diff, nil)
```

Comment:
This gives us an excellent entry point to sort `stables`. 😄

That should probably be a different PR.